src/sass/_base.scss

```diff
@@ -1,5 +1,117 @@
 * {
   box-sizing: border-box;
+  font-family: "Jetbrains Mono", sans-serif;
 }
 
-body {}
+body {
+  background-color: $background;
+  margin: 0;
+}
+
+section {
+  border: 1px solid $button;
```

Comment:
Seeing a variable name like this always makes me aware I should probably go back and rename my variable so it makes sense wherever I use it.